SamplifyAPIClient/SamplifyClient.cs

```diff
@@ -451,10 +477,18 @@ private bool IsProdEnvironment(SamplifyEnv env)
 
         internal static class HostConstants
         {
+            // PROD
             internal const string ProdAuthBaseURL = "https://api.Dynata.com/auth/v1";
             internal const string ProdAPIBaseURL = "https://api.Dynata.com/sample/v1";
+
+            // UAT
             internal const string UATAuthBaseURL = "https://api.uat.pe.Dynata.com/auth/v1";
             internal const string UATAPIBaseURL = "https://api.uat.pe.Dynata.com/sample/v1";
+
```

Comment:
lets not add dev environment in the sdk. This is a public sdk and external clients should not have access to our dev environment.